docs/blogs/dynamo-snapshot/dynamo-snapshot.md

```
@@ -0,0 +1,248 @@
+---
+
+# Dynamo Snapshot: Fast Startup for Inference Workloads on Kubernetes
+
+## The Cold-Start Problem
+The primary objective when optimizing LLM inference workloads is tokens per second — the higher the productive output per second of GPU time, the better the economics. Every second the GPU isn't producing tokens is money lost.
+
+But steady-state performance numbers only tell us part of the story. When serving inference workloads in production, there are many reasons why a workload cannot run in the same configuration forever: customer demand fluctuates, model preferences shift, configurations are updated to chase better numbers, and workloads themselves occasionally fail and have to be restarted. The ability to bring up new instances of inference engines for a particular model/configuration, even on a single GPU, can take on the order of minutes. That is a long time to be paying for GPUs that aren't producing any tokens.
+
+Here is a breakdown of the cold start time of various models for a single-GPU workload:
+
+![](./figures/cold_start_bench.svg)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)